ainstein_radar_drivers/nodes/wayv_node

```diff
@@ -576,28 +508,50 @@ class WayvROSInterface( object ):
                                 print("Present Radar Config: (",v.radar_config.cmd_count,"lines )\r\n")
                                 print("")
 
-                    radars_seen[v.id].publish_boundary_box()
-                    radars_seen[v.id].publish_wayv_config()
+                    self.radars_seen[v.id].publish_wayv_config()
             #Skip if a radar connecting causes a thread error with the dictionary
             except RuntimeError:
                 pass
             except KeyError:
                 pass
 
-            if len(query_config) > 0:
-                id = query_config.pop(0)
+            if len(self.query_config) > 0:
+                id = self.query_config.pop(0)
                 self.wayv_air.query_config(id)  # the API only support querying one radar at a time
-                query_config_requested.append(id)
+                self.query_config_requested.append(id)
 
-            elif len(pcl_config) > 0:
-                id = pcl_config.pop(0)
-                self.wayv_air.enable_pcl(id, enbl_pcl)
-                if id not in query_config:
-                    query_config.append(id)
+            elif len(self.pcl_config) > 0:
+                id = self.pcl_config.pop(0)
+                self.wayv_air.enable_pcl(id, self.enbl_pcl)
+                if id not in self.query_config:
+                    self.query_config.append(id)
 
             # Wait 2s to run supervisor task again
             time.sleep(2)
 
+    def determine_radar_label(self, new_radar: Wayv_Air_Radar) -> str:
+        label = new_radar.comm_config.dev_id
+        suffix = 1
+        if len(self.radars_seen) > 0:
+            for radar in self.radars_seen.items():
+                if label == radar[1].radar_label:
+                    label = label + "_" + str(suffix)
+                    suffix += 1
+        return label
+    
+    def shutdown(self):
+        self.run_supervisor = False
+        self.wayv_air.shutdown()
+        if self.logging_mode:
+            for file in self.logging_files.values():
+                file.close()
+
+
+def sigint_handler(*args):
+    wayv_intf.shutdown()
+    sys.stderr.write('/r')
+    sys.exit()
+
 if __name__ == "__main__":
     try:
         rospy.init_node( 'wayv_node' )
```

Comment:
Since you are cleaning up some legacy code, line 559, `data_port` is not used. Please remove.